travis-build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Cabal
+
+cabal new-build all --enable-tests
+cabal new-test all --enable-tests
+# git fetch origin master:origin/master && git rebase origin/master --exec "cabal new-build all --enable-tests"
```

Comment:
I think that given that now this PR is targeting master, this can be uncommented, and it should build correctly again.